cpp/ql/test/query-tests/Likely Bugs/Underspecified Functions/TooFewArguments.expected

```diff
@@ -1,4 +1,2 @@
-| test.c:34:3:34:19 | call to not_yet_declared2 | This call has fewer arguments than required by $@. | test.c:32:3:32:3 | not_yet_declared2 | not_yet_declared2 |
-| test.c:34:3:34:19 | call to not_yet_declared2 | This call has fewer arguments than required by $@. | test.c:76:6:76:22 | not_yet_declared2 | not_yet_declared2 |
```

Comment:
The test file probably needs to be updated to indicated that we do not detect these.